waspc/src/Generator/WebAppGenerator/OperationsGenerator.hs

```diff
@@ -60,8 +66,13 @@ genAction _ action = C.makeTemplateFD tmplFile dstFile (Just tmplData)
         , "actionRoute" .=
             (ServerGenerator.operationsRouteInRootRouter
              ++ "/" ++ ServerOperationsRoutesG.operationRouteInOperationsRouter operation)
+        , "entitiesArray" .= makeEntitiesJsArray operation
         ]
     operation = Wasp.Operation.ActionOp action
 
+makeEntitiesJsArray :: Wasp.Operation.Operation -> String
```

Comment:
maybe small description/example? Also maybe rename to `makeJsArrayWithEntities` or sth like that?

Reply:
Added comment with example and renamed to makeJsArrayOfEntityNames.